.github/README.md

```diff
@@ -1,164 +1,74 @@
+We have a lot of modules already made by the community, many of which can be found in the [Module Catalogue](https://www.azerothcore.org/catalogue.html#/).
 
 ## Installation
 
-Installation instructions are available [here](http://www.azerothcore.org/wiki/Installation).
+Detailed installation instructions are available [here](http://www.azerothcore.org/wiki/installation).
 
-We also have an auto-installation bash script [here](/apps/installer/main.sh) (*Warning: try it/analyze it before running it*).
-
-Dockerization of AzerothCore is fully supported, and we have various community-made tutorials (eg: AWS / Digital Ocean installation).
+## Contributing
 
+AzerothCore can also serve as a learning resource for aspiring developers who want to understand how WoW servers work, how MMORPGs are structured, how game server emulators are created, or to improve their C++ and SQL knowledge.
 
-## Support
+If you want to contribute to the project, you will find a lot of resources that will guide you in our [wiki](https://www.azerothcore.org/wiki/contribute).
 
-Our self-made wiki probably has a lot of answers for you.
+We also recommend you read our [CODE OF CONDUCT](https://github.com/azerothcore/azerothcore-wotlk/blob/master/.github/CODE_OF_CONDUCT.md).
 
-For help requests, it is recommended to ask your question on [StackOverflow](https://stackoverflow.com/questions/tagged/azerothcore) and link it in [our chat](https://discordapp.com/channels/217589275766685707/284406375495368704).
+Feel free to join our [Discord server](https://discord.gg/gkt4y2x).
 
+Click on the "⭐ Star" button to help us gain more visibility on Github!
 
 ## Authors & Contributors
 
-This project exists thanks to:
-
-- **The [AzerothCore developers and contributors](https://github.com/AzerothCore/azerothcore-wotlk/graphs/contributors)**
-- The [SunwellCore developers xinef and pussywizard](http://www.azerothcore.org/pages/sunwell.pl/)
-- All the [TrinityCore developers and contributors](https://github.com/TrinityCore/TrinityCore/blob/3.3.5/AUTHORS)
-- All the [MaNGOS, ScriptDev2 and UDB developers and contributors](https://github.com/cmangos/mangos-wotlk/blob/master/AUTHORS.md)
-
+This project exists thanks to the [AUTHORS](https://github.com/azerothcore/azerothcore-wotlk/blob/master/AUTHORS).
 
 ## Important Links
 
 - [Doxygen Documentation](https://www.azerothcore.org/pages/doxygen/index.html)
-
-- [Code of Conduct](https://github.com/azerothcore/azerothcore-wotlk/blob/master/.github/CODE_OF_CONDUCT.md)
 - [Website](http://www.azerothcore.org/)
 - [AzerothCore catalogue](http://www.azerothcore.org/catalogue.html  "Modules, tools, and other stuff for AzerothCore") (modules, tools, etc...)
-- [Module template / Module skeleton](https://github.com/azerothcore/skeleton-module/)
-- [Our community hub (Discord)](https://discord.gg/gkt4y2x)
+- [Our Discord server](https://discord.gg/gkt4y2x)
 - [Our wiki](http://www.azerothcore.org/wiki "Easy to use and developed by AzerothCore founder")
```

Comment:
I know this wasn't an addition on your part, but why are there statements in these hyperlinks that aren't visible to a reader?

Reply:
they are visible if you hover over them. I guess that's why they were originally added

Reply:
> they are visible if you hover over them. I guess that's why they were originally added

Learn something new every day, I guess. I don't mind 'em then.